user-guide/modules/ROOT/pages/boost-test-tutorial.adoc

```diff
@@ -13,11 +13,11 @@ Let's get started with a trivial example.
 
 Here's a very simple example of a test suite with Boost.Test:
 
-. Use Visual Studio to create a C++ Console application, call it something like "StringRev".
+. Use Visual Studio to create a C++ Console application, call it something like "Trivia".
 
-. In the project *Properties* for *C++/General*, locate *Additional Include Directories* and add the path to your Boost libraries. The path will be something like "C:\Users\Peter\Documents\Boost\boost_1_81_0".
+. In the project *Properties* for *C++/General*, locate *Additional Include Directories* and add the path to your Boost libraries. The path will be something like "C:\Users\<your path>\boost_1_81_0".
 
-. Then, still in *Properties*, but now for *Linker/General* add to the *Additional Library Directories* with the path to your Boost lib folder. This path will be something like "C:\Users\Peter\Documents\Boost\lib".
+. Then, still in *Properties*, but now for *Linker/General* add to the *Additional Library Directories* with the path to your Boost lib folder. This path will be something like "C:\Users\<your path>\lib".
```

Comment:
I believe the convention for paths is using "`", just like we do for code.

That is, `C:\Users\Peter\Documents\Boost\lib` (with "`" as quotes) instead of "C:\Users\Peter\Documents\Boost\lib".

Reply:
@alandefreitas OK, will change this.